src/dronevis/tracking/human_tracking.py

```diff
@@ -0,0 +1,165 @@
+        """
+        Load YOLO v4 model using the configration file and weights attached in tracker folder
+        Initialize DeepSORT tracker
+        """
+        from deep_sort_realtime.deepsort_tracker import DeepSort
+        yolo_cfg_path = find ("yolov4.cfg")
+        if yolo_cfg_path == None:
+            print("Downloading Faster YOLO v4 configuration...")
+            yolo_cfg_path, headers = urllib.request.urlretrieve(
+                "https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v3_optimal/yolov4.cfg", 
+                filename= os.path.join(os.getcwd(),"yolov4.cfg"))
+        
+        yolo_weights_path = find ("yolov4.weights")
+        if yolo_weights_path == None:
+            print("Downloading Faster YOLO v4 weights...")
+            yolo_weights_path, headers = urllib.request.urlretrieve(
+                "https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v3_optimal/yolov4.weights", 
+                filename= os.path.join(os.getcwd(),"yolov4.weights"))
+        print("Loading Faster YOLO v4 model ...")
+        self.model = cv2.dnn.readNet(yolo_cfg_path, yolo_weights_path)
+        self.tracker = DeepSort()
+
+
+    def transform_img(self, img: np.ndarray) -> torch.Tensor:
+        """Transform image to tensor
+
+        Args:
+            img (numpy.ndarray): input array
+
+        Returns:
+            torch.Tensor: tensor img
+        """
+        assert self.transform is not None, "Model not initialized. You need to load the model first. Please run `load_model`."
+        return self.transform(to_pil_image(img)).to(self.device)
+
+    def predict(self,image:np.ndarray) -> list:
+        """
+        Detect Humans in an image using YOLO v4
+        Args:
+            image (np.ndarray): input image
+        Returns:
+            List of Detections, where each detection is represented as a tuple
+            of bounding box coordinates, classid and score
+        """
+        detections = []
+        model = cv2.dnn_DetectionModel(self.model)
+        model.setInputParams(size=(416, 416), scale=1/255, swapRB=True)
+        classes, scores, boxes = model.detect(image, self.CONFIDENCE_THRESHOLD, self.NMS_THRESHOLD)
+        for (classid, score, box) in zip(classes, scores, boxes):
+            label = self.class_names[classid[0]]
+            if (label == "person"):
+                detections.append((box,classid,score))
+        return detections
+    
+
+    def detect_webcam(self,video_index: Union[str, int] = 0, window_name: str = "Cam Detection") -> None:
+        """Start webcam detection from video_index
+        *(to quit running this function press 'q')*
+        Args:
+            video_index (Union[str, int], optional): index of video stream device. Defaults to 0 (webcam).
+            window_name (str, optional): name of cv2 window. Defaults to "Cam Detection".
+        """
+        cap = cv2.VideoCapture(video_index)
+        prev_time = 0

```

Comment:
A bit nit-picky here, but since variable is used below  as a `float`, it is a good idea to initialize it as a float. `prev_time = 0.0`